public/assets/images/navigation/HomeLogoSelected.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface Prop {
+  color: string;
+}
+function HomeLogoSelected(prop: Prop) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 255.12 255.12"
+      width="24"
```

Comment:
I'd suggest checking out the other navigation icons. They differ in viewbox (most have a height of 512 defined in the viewbox i.e. viewBox="0 0 widthValue 512), and no height/width is explicitly provided. Not defining dimensions within this component itself makes the dimensions easier to manipulate via code (if needed).